test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss_alpha.yaml

```diff
@@ -0,0 +1,55 @@
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-pd-controller
+spec:
+  serviceName: "csi-gce-pd"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gcp-compute-persistent-disk-csi-driver
+  template:
+    metadata:
+      labels:
+        app: gcp-compute-persistent-disk-csi-driver
+    spec:
+      serviceAccountName: csi-controller-sa
+      containers:
+        - name: csi-provisioner
+          image: gcr.io/gke-release/csi-provisioner:v1.0.0-gke.0
+          args:
+            - "--v=5"
+            - "--provisioner=pd.csi.storage.gke.io"
```

Comment:
@davidz627 can we finally get rid of this field? I think v1.0.0+ should include this: https://github.com/kubernetes/kubernetes/pull/71537

Reply:
wrong link?
but yea I think we should be able to get rid of it, if we finally cherrypicked that change to release-1.0 branch